Sources/BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -252,10 +253,7 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
 #ifdef __BT_APPLE_PAY
             BTJSON *applePayConfiguration = self.configuration.json[@"applePay"];
             if ([applePayConfiguration[@"status"] isString] && ![[applePayConfiguration[@"status"] asString] isEqualToString:@"off"] && !self.dropInRequest.applePayDisabled && self.configuration.canMakeApplePayPayments) {
-                // Short-circuits if BraintreeApplePay is not available at runtime
-                if (__BT_AVAILABLE(@"BTApplePayClient")) {
```

Comment:
We were seeing an error with `__BT_AVAILABLE`, and since this check is doing the same thing as `#ifdef __BT_APPLE_PAY` (line 252), we decided to remove it.